mne_bids/mne_bids.py

```diff
@@ -328,14 +396,18 @@ def raw_to_bids(subject_id, task, raw_file, output_path, session_id=None,
         The session name in BIDS compatible format.
     run : int | None
         The run number for this dataset.
-    kind : str, one of ('meg', 'ieeg')
+    kind : str, one of ('meg', 'eeg', 'ieeg')
         The kind of data being converted. Defaults to "meg".
     events_data : str | array | None
         The events file. If a string, a path to the events file. If an array,
         the MNE events array (shape n_events, 3). If None, events will be
-        inferred from the stim channel using `find_events`.
-    event_id : dict
-        The event id dict
+        inferred from the stim channel using `mne.find_events`.
+    event_id : dict | None
+        The event id dict used to create a 'trial_type' column in events.tsv
+    event_type : dict | None
```

Comment:
same question as before. Which field is this?

Reply:
see above, "event_type" is a new BIDS EEG field to describe event_values (=event_id) in case the description is different from "trial_type". Generally, you can think about it this way: If we have an experimental trial, there might be several trigger codes nested in that trial: one signifying a button press, another signifying the onset of an additional stimulation or feedback, etc. --> these things can go into "event_type", while "event_id" covers the BIDS trial_type column (previously "condition" in this code)

Reply:
What you mention could perhaps also be a solution. I am trying to stick to BEP_006 so let me try to argue --> I think that the event_type column might be used for really low level things like e.g.: Marker 1 signifies onset of a "famous face" trial then there is some jitter before the actual stimulus is shown, this could be Marker 2. --> for the "trial_type" column, which is described via event_id, these both markers get the entry "famous face" ... but in the "event_type" column, these might have the entries "trial onset" and "face_show_onset"

Reply:
I guess there is still scope to change BEP_006 :) That's why it's a good idea to get your hands dirty to see if the standard makes sense in practice.

Anyway, it feels a bit like overengineering to me. If it's an optional field, maybe we could leave it out for now and address it in a future PR?

Reply:
if you insist, I'll remove it! :-) BEP_006 is unlikely to change, we're kind of wrapping things up now

Reply:
yeah, I'd like to keep things as simple and as automated as possible. I just noticed that there is also a field called `event_value`, which you can probably update automatically :-)

Reply:
okay, I'll remove it - `event_value` is inferred from mne's `events`